setup.py

```diff
@@ -113,7 +113,7 @@ def run(self):
 
 install_requires = [
     'bokeh >=3.4.0,<3.5.0',
-    'param >=2.0.0,<3.0',
+    'param >=2.1.0,<3.0',
```

Comment:
Are you sure that we want to bump versions in a micro release?

Reply:
I would argue bumping versions to resolve bugs is appropriate in a patch release, but I'm open to alternative opinions.

Reply:
My feeling is that if it were a bug in a 3rd party library like Pandas, we would not bump the version. We would likely do a version check, and accept the shortcomings if you are not using a version supporting the bug.

Reply:
Depends on the severity of the bug I suppose. If it was just this bug I'd definitely postpone the bump but there's a variety of other bugs that are more far reaching. Really should have bumped in 1.4.0 already.